components/tidb_query_vec_expr/src/impl_compare.rs

```diff
@@ -883,8 +889,8 @@ mod tests {
     }
 
     #[test]
-    fn test_greatest_int() {
-        let cases = vec![
+    fn test_greatest() {
```

Comment:
It doesn't seem that putting both int and real cases inside the same test saves code. Could you put them in separate tests?

Reply:
Done